src/aap_eda/settings/default.py

```diff
@@ -654,29 +653,17 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
     "SECRET_KEY": settings.get("RESOURCE_SERVER__SECRET_KEY", ""),
     "VALIDATE_HTTPS": settings.get("RESOURCE_SERVER__VALIDATE_HTTPS", False),
 }
-RESOURCE_JWT_USER_ID = settings.get("RESOURCE_JWT_USER_ID", "")
-RESOURCE_SERVICE_PATH = settings.get(
-    "RESOURCE_SERVICE_PATH", "/api/gateway/v1/service-index/"
-)
-
-if DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED:
-    if (
-        RESOURCE_SERVER["URL"]
-        and RESOURCE_SERVER["SECRET_KEY"]
-        and RESOURCE_JWT_USER_ID
-    ):
-        RQ_PERIODIC_JOBS.append(
-            {
-                "func": "aap_eda.tasks.shared_resources.resync_shared_resources",  # noqa E501
-                "interval": 900,
-                "id": "resync_shared_resources",
-            }
-        )
-    else:
-        raise ImproperlyConfigured(
-            "RESOURCE_SERVER__URL, RESOURCE_SERVER__SECRET_KEY, "
-            "and RESOURCE_JWT_USER_ID settings must be properly configured"
-        )
+RESOURCE_JWT_USER_ID = settings.get("RESOURCE_JWT_USER_ID", None)
+RESOURCE_SERVICE_PATH = settings.get("RESOURCE_SERVICE_PATH", None)
+
+if RESOURCE_SERVER["URL"] and RESOURCE_SERVER["SECRET_KEY"]:
+    RQ_PERIODIC_JOBS.append(
+        {
+            "func": "aap_eda.tasks.shared_resources.resync_shared_resources",
+            "interval": 900,
+            "id": "resync_shared_resources",
+        }
+    )
```

Comment:
@bzwei Should we set ALLOW_LOCAL_RESOURCE_MANAGEMENT to False here? Which would mean that ALLOW_LOCAL_RESOURCE_MANAGEMENT is a setting that is controlled by the presence of RESOURCE_SERVER["URL"]
Do we also need to check if the RESOURCE_SERVER is set?

Reply:
@rochacbruno's previous commented not to depend on ALLOW_LOCAL_RESOURCE_MANAGEMENT. How do we check RESOURCE_SERVER since it is a dict with keys already. Here thus I directly check the keys.

Reply:
Those 2 variables are meant to be independent.

- `RESOURCE_SERVER` - dict of settings to authenticate to the server
- `ALLOW_LOCAL_RESOURCE_MANAGEMENT` - Flag indicating if local service can mutate local settings or only RESOURCE_Server can do it.